src/main/kotlin/com/pk/Main.kt

```diff
@@ -65,10 +65,18 @@ fun main(args: Array<String>) = mainBody {
             USE_ARGPARSER to useArgParser
         )
 
+        val defaultDependencies = mutableListOf<Dependency>()
+
+        if (useArgParser) {
+            defaultDependencies.add(Dependency("implementation", "com.xenomachina", "kotlin-argparser"))
+        }
+
+        defaultDependencies.addAll(dependencies())
+
```

Comment:
Maybe there's a better (more idiomatic?) kotlin way to do this with union() and immutable collections, but I didn't spend too much time trying to figure it out...